robot/Cumulus/tests/browser/gift_entry/lookup_related_fields_validation_for_payment.robot

```diff
@@ -0,0 +1,154 @@
+*** Settings ***
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             API Check And Enable Gift Entry
+Suite Teardown  Run Keywords
+...             Query And Store Records To Delete    ${NS}DataImport__c   ${NS}NPSP_Data_Import_Batch__c=${BATCH_Id}
+...   AND       Capture Screenshot and Delete Records and Close Browser
+
+*** Variables ***
+&{PAYMENT_REF}             Default Value=123abc
+&{PAYMENT_METHOD}          Default Value=Cash
+&{PAID}                    Default Value=False
+
+*** Keywords ***
+Setup Test Data
+    [Documentation]       Create contact, opportunity, get current date and npsp
+    ...                   generate random names for template and batch.
+
+    &{CONTACT} =          API Create Contact       FirstName=${faker.first_name()}    LastName=${faker.last_name()}
+    Set suite variable    &{CONTACT}
+    Store Session Record  Account                  ${CONTACT}[AccountId]
+    ${DATE} =             Get Current Date         result_format=%Y-%m-%d
+    Set suite variable    ${DATE}
+    &{OPPORTUNITY} =      API Create Opportunity   ${CONTACT}[AccountId]           Donation
+    ...                   StageName=Pledged
+    ...                   Amount=200
+    ...                   CloseDate=${DATE}
+    ...                   Name=${CONTACT}[Name] Donation
+    Set suite variable    &{OPPORTUNITY}
+    &{PAYMENT1} =         API Create Payment            ${OPPORTUNITY}[Id]
+    ...                                  npe01__Payment_Amount__c=100.0
+    ...                                  npe01__Scheduled_Date__c=${DATE}
+    Set suite variable    &{PAYMENT1}
+    &{PAYMENT2} =         API Create Payment            ${OPPORTUNITY}[Id]
+    ...                                  npe01__Payment_Amount__c=300.0
+    ...                                  npe01__Scheduled_Date__c=${DATE}
+    Set suite variable    &{PAYMENT2}
+    ${TEMPLATE_NAME} =    Generate Random String
+	Set suite variable    ${TEMPLATE_NAME}
+    ${BATCH_NAME} =       Generate Random String
+	Set suite variable    ${BATCH_NAME}
```

Comment:
alignment seems a little off

Reply:
they are aligned in the robot test case, they look not aligned here at GitHub UI